### PR TITLE
api: promote to official APIs.

### PR DESCRIPTION
**Pull request by hermet**
New API:
- SwCanvas::Colorspace::ABGR8888S
- SwCanvas::ColorSpace::ARGB8888S

### CODE REVIEW

**Comment by hermet** [comment]
Note: CanvasEngine::All will not be in v0.12.x

File: inc/thorvg.h [hunk 1]
```diff
@@ -212,7 +212,7 @@ enum class BlendMethod : uint8_t
  */
 enum class CanvasEngine : uint8_t
 {
-    All = 0,       ///< All feasible rasterizers.
+    All = 0,       ///< All feasible rasterizers. @since 1.0
     Sw = (1 << 1), ///< CPU rasterizer.
     Gl = (1 << 2), ///< OpenGL rasterizer.
```
Comment (RuiwenTang) [inline inc/thorvg.h:217]: does GL backend needs to be marked as experimetal API ?

Comment (hermet) [reply inc/thorvg.h:217]: @RuiwenTang It's been already there from the beginning. Instead, we should not change the enum value.  GlCanvas has that experimental tag so It think it wouldn't be a problem.

Comment (hermet) [reply inc/thorvg.h:217]: @RuiwenTang fyi, "All" was newly added since 1.11.

**Review by SergeyLebedkin — APPROVED** [review]

**Review by tinyjin — APPROVED** [review]

**Pull request merged**